.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -69,15 +86,20 @@ jobs:
       - name: Install npm dependencies
         run: npm ci
       - name: Building standalone plugins
+        if: ${{ matrix.directory != 'plugins' }}
         run: npm run build-plugins
```

Comment:
The language here is now confusing because the name is "Building standalone plugins" but in only runs when we're _not_ in the plugins directory. Can we remove this entirely in this branch, since we're going to be moving all of the standalone plugins to the plugins directory anyway?

Reply:
Yes, but for now, we need plugins from the build, so I have excluded these steps for the plugin, as we don't need a build process for it.

- Build - https://github.com/WordPress/performance/actions/runs/8015338254/job/21895352814?pr=1000
- Plugin - https://github.com/WordPress/performance/actions/runs/8015338254/job/21895352942?pr=1000